packages/shared/src/utils/shallowCompare.ts

```diff
@@ -0,0 +1,14 @@
+export function shallowCompare(obj1: any, obj2: any) {
```

Comment:
May moving it to `object.ts` is better? The utils files are classified by their service objects.